index.html

```diff
@@ -0,0 +1,451 @@
+<!doctype html>
+<script src=/boot.js></script>
+<title>Composed Schema Definition Layer</title>
+
+<div id=view>
+<main markdown>
+
+<header>
+  <h1>Composed Schema Definition Layer</h1>
+  <h2>schema elements for describing composed subgraphs</h2>
+  <table class=spec-md>
+    <tr>
+      <td>status</td>
+      <td id=spec-md-status>Draft</dd>
+    </tr>
+    <tr>
+      <td>version</td>
+      <td id=spec-md-version>0.1</dd>
+    </tr>
+    <tr>
+      <td>authors</td>
+      <td id=spec-md-authors>
+        <ul class=spec-authors-list>
+          <li><a rel=author href=mailto:jake@apollographql.com>Jake Dawkins</a>
+          <li><a rel=author href=mailto:trevor@apollographql.com>Trevor Scheer</a>
+          <li><a rel=author href=mailto:ashi@apollographql.com>Ashi Krishnan</a>
+        </ul>
+      </td>
+    </tr>
+  </table>
+</header>
+
+## Introduction: What is CSDL?
+
+```mermaid
+#::[Illustration: CSDL composes multiple subgraph schemas.]
+graph LR
+  s1(astronauts.graphql)-->CSDL
+  s2(rockets.graphql)-->CSDL
+  s3(robots.graphql)-->CSDL
+  style CSDL fill:none,stroke:fuchsia,color:fuchsia;
+```
+
+The Composed Schema Definition Layer (CSDL) provides schema elements to describe a GraphQL Schema which is composed from one or more other GraphQL schemas. **Schema elements** comprise any item in a GraphQL schema—types, input types, directives, etc. CSDL defines directives and mandatory input types. 
+
+The CSDL is intended to be a single artifact which can power a graph router. It declares the composed schema and annotates it with directives that allow the router to disassemble input queries and dispatch them to subgraphs.
+
+CSDL restricts the [GraphQL interface description language](https://spec.graphql.org/) in certain ways: type extensions are not permitted, for instance. It also requires the definition of certain types and provides a suite of directives to describe graph topology. In particular, a CSDL can:
+- define [subgraphs](#subgraph) and bind them to [endpoints](#endpoint) (with the required [`csdl_Graph`](#csdl_Graph) type and [`@csdl_endpoint`](#csdl_endpoint) directive)
```

Comment:
These links don't work (similarly for other references).

Reply:
fixed